src/pages/signin/LoginForm.js

```diff
@@ -82,6 +85,13 @@ class LoginForm extends React.Component {
     }
 
     componentDidMount() {
+        // eslint-disable-next-line rulesdir/prefer-early-return
+        this.unsubscribeVisibilityListener = Visibility.onVisibilityChange(() => {
```

Comment:
It's an interesting question, but I don't think it would be necessary because most of the inputs are contained within forms where a user can select any one of them at any time when they switch to a different screen and back. This is kept simple because it's only a single input which the user can keep track of.

Additionally, implementing this for all the inputs would either require modifying every single page that contains a form, which would take a lot of time to test properly, or implementing it at the TextInput component level, which could result in a performance hit because visibility would be checked for every single input in a form, for example.

Reply:
>Should we do this change for all inputs in the app?

I think it would be nice to have this for all inputs @akinwale @parasharrajat, but can't we have this implemented in Form instead? I think every form is wrapped in it except for LoginForm and maybe a few others. But the majority of forms I believe are wrapped in our Form component. If we can have this implemented there then I'd say let's go for it!

Reply:
@youssef-lr I think it's best to do this at the `TextInput` level, since as you pointed out, some of the inputs are not wrapped with the `Form` component, and doing this in the `Form` would miss all of those inputs.

@parasharrajat What are you thoughts on this?

Reply:
At `TextInput` level sounds good to me!

Reply:
Done!